Products/ZenModel/MibOrganizer.py

```diff
@@ -227,7 +229,7 @@ def handleUploadedFile(self, REQUEST):
         """
         filename = REQUEST.upload.filename
         mibs = REQUEST.upload.read()
-        savedMIBPath = zenPath('/var/ext/uploadedMIBs/%s' % filename)
+        savedMIBPath = zenPath(_pathToMIB'/%s' % filename)
```

Comment:
This is a syntax error, but you should be able to write the `zenPath` call like this:
```
savedMIBPath = zenPath(_pathToMIB, filename)
```
`zenPath` behaves like `os.path.join` with regard to multiple arguments.